index.html

```diff
@@ -829,246 +829,192 @@ <h4>
+              </tr>
+              <tr>
+                <td>七号</td>
+                <td>5.25pt</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <p its-locale-filter-list="zh-hant" lang="zh-hant">中文活字大小有不同單位。在金屬活字時代，傳統中文活字尺寸以「號」為單位，故稱作「字號」；在照相排版時代沿用照排機尺寸的單位「級」，故稱作「字級」；在桌面排版時代，直接使用桌面排版軟件中的「點」（DTP point）。目前，很多場合的中文排版依舊習慣沿用「號」。</p>
+          <p its-locale-filter-list="zh-hant" lang="zh-hant">「號」由於當年金屬活字各地廠家的規範不一而不盡相同，「號」也有英美、歐陸、DTP等多種制式，導致「號」與「點」的換算有不同方法。下表僅列出常見的一些換算數值，僅供參考，不作為規範性規定：</p>
+          <table class="charsize" its-locale-filter-list="zh-hant" lang="zh-hant">
+            <thead>
+              <tr>
+                <th>號數</th>
+                <th>點數</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>初號</td>
+                <td>42 pt</td>
+              </tr>
+              <tr>
+                <td>一號</td>
+                <td>27.5/28 pt</td>
+              </tr>
+              <tr>
+                <td>小（新）一號</td>
+                <td>24 pt</td>
+              </tr>
+              <tr>
+                <td>二號</td>
+                <td>21/22 pt</td>
+              </tr>
+              <tr>
+                <td>小（新）二號</td>
+                <td>18 pt</td>
+              </tr>
+              <tr>
+                <td>三號</td>
+                <td>15.75/16 pt</td>
+              </tr>
+              <tr>
+                <td>四號</td>
+                <td>13.75/14 pt</td>
+              </tr>
+              <tr>
+                <td>小（新）四號</td>
+                <td>12 pt</td>
+              </tr>
+              <tr>
+                <td>五號</td>
+                <td>10.5 pt</td>
+              </tr>
+              <tr>
+                <td>小（新）五號</td>
+                <td>9 pt</td>
+              </tr>
+              <tr>
+                <td>六號</td>
+                <td>7.875/8 pt</td>
+              </tr>
+              <tr>
+                <td>七號</td>
+                <td>5.25pt</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <p its-locale-filter-list="en" lang="en">Size 5 is usually used for text content. Newspapers and magazines use both Size 5 and New Size 5. The acceptable minimum size for the text in content is Size 6  (7.875pt≒2.8mm). If a smaller size is used, it will be difficult to read due to the complex structure of Chinese characters.</p>
+          <p its-locale-filter-list="zh-hans" lang="zh-hans">一般内文主要使用五号字（10.5pt≒3.7mm），而报纸、杂志则使用新五号字（9pt≒3.2mm），两种皆常用。而一般内文字最小使用到六号字（7.875pt≒2.8mm），若小于此尺寸，由于汉字结构复杂，较难阅读。</p>
+          <p its-locale-filter-list="zh-hant" lang="zh-hant">一般內文主要使用五號字（10.5pt≒3.7mm），而報紙、雜誌則使用新五號字（9pt≒3.2mm），兩種皆常用。而一般內文字最小使用到六號字（7.875pt≒2.8mm），若小於此尺寸，由於漢字結構複雜，較難閱讀。</p>
+        </li>
         <li id="id52">
           <p its-locale-filter-list="en" lang="en">Line length should be multiples of the character size and each line should be in alignment with one another. </p>
```

Comment:
I'm not entirely clear about the meaning of "each line should be in alignment with one another" - does it mean that characters should be aligned, or line ends, etc?

Reply:
I think it means the line ends should be aligned.

Reply:
Edited.